x-pack/plugins/canvas/i18n/ui.ts

```diff
@@ -234,6 +234,10 @@ export const ArgumentStrings = {
       i18n.translate('xpack.canvas.uis.arguments.paletteLabel', {
         defaultMessage: 'Choose a color palette',
```

Comment:
Wondering if we can drop this tooltip altogether, as the form label alone seems to be sufficient from a usability perspective. Thoughts?